addons/sale/models/sale_order_line.py

```diff
@@ -867,6 +867,13 @@ def _onchange_product_packaging_id(self):
                     },
                 }
 
+    def update_all_lines(self, field_name, value):
+        self.ensure_one()
+        self.order_id.order_line.update({field_name: value})
```

Comment:
The ORM should decide what's the best between write/update.

```suggestion
        self.order_id.order_line[field_name] = value
```

Are there other use case than just updating the discount ?